mod/05_email/set/right/bcc.rb

```diff
@@ -18,7 +22,7 @@ def process_email_addresses context_card, format_args, args
 format :email_text do
   view :email_addresses do |args|
     context = args[:context] || self
-    card.item_names.map do |item_name|
+    card.item_names(:context=>context.cardname).map do |item_name|
```

Comment:
Agreed.  This can be fixed so it doesn't need that line, but fine for now.  Maybe drop in a TODO comment.